tests/samples/test_api.py

```diff
@@ -930,15 +930,20 @@ async def test_analyze(
     spawn_client,
     static_time,
     resp_is,
-    test_random_alphanumeric,
 ):
     mocker.patch("virtool.samples.utils.get_sample_rights", return_value=(True, True))
 
     client = await spawn_client(authorize=True)
     client.app["jobs"] = MockJobInterface()
 
     if error != "400_reference":
-        await client.db.references.insert_one({"_id": "foo"})
+        await client.db.references.insert_one(
+            {
+                "_id": "foo",
+                "name": "foo",
+                "data_type": "genome",
+            }
+        )
```

Comment:
Please use realistic names as described for the data layer tests.

It makes it easier to scan over the snapshots if they change.